### ISSUE TITLE
Allow setting keybinding to focus the chat terminal and to focus /expand last chat terminal output

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#274411]
cc @jooyoungseo, @rperez030 

Currently, this action is not registered, so a user cannot set this

https://github.com/microsoft/vscode/blob/b770d1f0fa2dbc440735e7b9e1d67e4257f8c06a/src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolProgressPart.ts#L527-L541

**Comment by meganrogge** [comment]
To verify, try these:

<img width="698" height="175" alt="Image" src="https://github.com/user-attachments/assets/447ad687-a64e-41e1-88ea-d002502a7897" />

**Comment by meganrogge** [comment]
This did not work and should be reverted. https://github.com/microsoft/vscode/pull/281029

This might be a timing issue as this works in OSS, not insider's. When I debugged, I saw that `_mostRecentProgressPart` was unexpectedly undefined which causes this bug.
